lib/decidim/reporting_proposals/config.rb

```diff
@@ -44,5 +44,10 @@ module ReportingProposals
     config_accessor :allow_proposal_photo_editing do
       true
     end
+
+    # Public setting to allow to assign other valuators
+    config_accessor :allow_to_assign_other_valuators do
```

Comment:
```suggestion
    config_accessor :valuators_assign_other_valuators do
```

Let's rename it like this, it's more clear